anime_downloader/sites/helpers/request.py

```diff
@@ -64,12 +78,14 @@ def setup_func(url: str,
                            url,
                            headers=default_headers,
                            **kwargs)
-        res.raise_for_status()
-        logger.debug(res.url)
-        # logger.debug(res.text)
+
+        if sess != selescrape: #TODO fix this for selescrape too
+            res.raise_for_status()
+            logger.debug(res.url)
         if logger.getEffectiveLevel() == logging.DEBUG:
```

Comment:
move this down to avoid errors. E.g.
```
        if sess != selescrape: """TODO fix this for selescrape too"""
            res.raise_for_status()
            logger.debug(res.url)
            if logger.getEffectiveLevel() == logging.DEBUG:
                _log_response_body(res)
```